### PR TITLE
Add competition detail screen

### PR DESCRIPTION
**Pull request by Priva28**

### CODE REVIEW

File: Project SF/Views/Tabs/Competitions/CompetitionsView.swift [hunk 1]
```diff
@@ -12,6 +12,8 @@
     var name: String
     var startDate: Date
     var endDate: Date
+    var points: Int = 0
+    var place: Int = 1
```
Comment (RomanEsin) [inline Project SF/Views/Tabs/Competitions/CompetitionsView.swift]: No need to explicitly define the type here, it'll be Int by default

**Review by RomanEsin — APPROVED** [review]

**Comment by Priva28** [comment]
I think this is ready to merge. If there are any changes/suggestions/bugs just lmk and I'd be happy to change stuff.

**Comment by Priva28** [comment]
Hmmm built fine for me. Can someone else check this?

**Comment by Priva28** [comment]
> Hmmm built fine for me. Can someone else check this?

I think it's using Xcode 11 build. I am using if statement within the main body which is only supported on Xcode 12/new swift version.

**Pull request merged**